build/configs/rtl8721csm/rtl8721csm_make_bin.sh

```diff
@@ -16,6 +17,10 @@
 #
 ###########################################################################
 
+TOP_PATH=/root/tizenrt/
```

Comment:
Is this working on each developer's env?

Reply:
I fixed this to make it more generic.